Sources/MapboxDirectionsCLI/CodingOperation.swift

```diff
@@ -81,6 +85,44 @@ class CodingOperation<ResponceType : Codable, OptionsType : DirectionsOptions >
         return interpolatedCoordinates
     }
     
+    private func requestResponse(_ coordinates: [Waypoint]?, includesSteps: Bool?) -> (Data?, Data) {
+        let semaphore = DispatchSemaphore(value: 0)
+        
+        guard let waypoints = coordinates, let includesSteps = includesSteps else {
+            print("Failed to request response without coordinates.")
+            exit(1) }
+        
+        let options = RouteOptions(waypoints: waypoints, profileIdentifier: .automobileAvoidingTraffic)
+        options.includesSteps = includesSteps
+        options.allowsUTurnAtWaypoint = true
+        options.includesAlternativeRoutes = false
+        options.includesExitRoundaboutManeuver = true
+        options.roadClassesToAvoid = []
+        options.routeShapeResolution = .full
```

Comment:
These settings should come from decoding the configuration file as a RouteOptions object.